### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ColeNelson806**
## Describe Your Changes
Updated README for better instructions on setting up the database.

## Type of Change
Please delete options that are not relevant.
- [x] Documentation update

## Checklist (DOD)
- [x] The code is written and passes all Acceptance criteria's
- [x] Any Zombie code is removed
- [x] Code has proper comment documentation
- [x] The code is unit tested with happy case and passes
- [x] Code is refactored for optimization
- [ ] The code has been peer-reviewed by one or more team members who did not work on it.
- [x] The code has no merge conflicts

### CODE REVIEW

**Review by GivenTeacher5 — CHANGES_REQUESTED** [review]
There were a few places during the database creation where it says "click on database" or something along those lines. My first thought was to look at the "Databases" tab, so maybe clarify on that. Then, there was the part for checking if the tables existed in the tables tab, but I didn't know where to find them.

**Review by Scott-Code-19 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -75,16 +81,18 @@ This application will be responsible for creating a User Interface that will all
...
+- Right-click the rules-engine database from the new server -> Query Tool
 - Paste in code from [sql/create.ddl](https://github.com/Access-Point-Program/cohort-3-rules-engine-admin/blob/2fa165b962dda1049bb8ee32e7312352060c6410/sql/create.ddl) file
 - Click play button  (Execute/Refresh)
 - Right-click database -> Refresh
-- Verify that ruleset, rule, and condition table are under the Tables tab.
-- Verify that eventtype, facttype and valuetype are under the Types tab. 
+- Click to open the rules-engine database you created, inside the new server
+- Click to open the "Schema" tab
+- Under the Tables tab, verify that ruleset, rule, and condition tables exist.
+- Under the Tables tab, verify that eventtype, facttype and valuetype exist.
```
Comment (sophialhales) [inline README.md:93]: update to Types

**Review by sophialhales — APPROVED** [review]
Looks good!

**Review by Scott-Code-19 — APPROVED** [review]
Looks GOOOODDDDDDDDDDDDDD👍

**Review by GivenTeacher5 — APPROVED** [review]
😎👍

**Pull request merged**